helm/README.md

```diff
@@ -0,0 +1,217 @@
+# Prometheus
+
+An opinionated [Prometheus](https://prometheus.io/) Helm chart for deployment in Giant Swarm clusters.
```

Comment:
Can we add that this is based off of upstream stable chart or so, we should give credit

Reply:
done in the bottom